### ISSUE TITLE
SNK Arcade Classics Vol. 1 for PS2

### ISSUE THREAD

**Issue by kjetil-f** [farmerbb/RED-Project#111]
Is it possible to extract the Neo Geo games from this compilation? It's possible with the Sega Genesis Collection, Atari Anthology, Activision Anthology and Intellivision Lives!.

**Comment by RealRelativeEase** [comment]
Not the PS2 version, but I've poked around inside the Wii version's files. The compilation uses Terminal Reality's .pod file format and it seems like audio (both music and sound effects) are stored in separate files compared to the "main" ROM. ng1rom.POD could be the Neo Geo BIOS? There are also fairly large POD files for KOF95-98, despite not being part of the compilation.

Unfortunately, I haven't been able to extract those files or do anything else with them.

**Comment by kjetil-f** [comment]
Thank you for the input. My guess is that the Wii and PS2 probably uses the same files since they were released the same year.

**Comment by RealRelativeEase** [comment]
I've tried to examine the .pod files inside SNK Arcade Classics Vol. 1 using Dragon UnPACKer 5, but the program crashes every time I try to open one of the files.

The same developer also did the Metal Slug Anthology, and from the look of things, it also uses Neo Geo ROMs for Metal Slug 1-5 (including Metal Slug X). The ROMs are stored in separate folders inside of `SLUGROM.POD`, along with what seems to be the BIOS. But once again, Dragon UnPACKer crashes every time I try to extract the contents of the folder. And from what I can tell, the ROM files seem to be stored in a similar fashion to the Neo Geo ROM ports by Code Mystics. #115

**Comment by kjetil-f** [comment]
Tested UnPACKer myself, but it would not open. Seems that you can only use [supported games](https://www.elberethzone.net/en/dragon-unpacker-supported-games.html) with this tool.

**Comment by RealRelativeEase** [comment]
I've been able to extract the contents of `SLUGROM.POD` using the [POD3 script](https://aluigi.altervista.org/bms/xentax_cs/POD_POD3.bms) for quickbms, but as expected, the folders can't just be zipped to play the respective Metal Slug game in an emulator.

For instance, this is what the first Metal Slug looks like: 
![mslug](https://github.com/farmerbb/RED-Project/assets/99961958/e0f8d00f-9196-49b1-a32e-22e6ac5962c6)

**Comment by kjetil-f** [comment]
Maybe there is a way to convert them. It doesn't look to different from the GOG version after all:

![msluggog](https://github.com/farmerbb/RED-Project/assets/93976052/f5c78af7-0226-406b-b4db-a70afc9b9070)

**Comment by RealRelativeEase** [comment]
I've tried to compare the files against the Neo Geo ROM from Humble Bundle (also the DotEmu version, so it should be identical to GOG). 201-m1.bin, 201-p1.bin, 201-v1.bin, 201-v2.bin are a match for the respective files in the DotEmu ROM according to HxD, even before the file extension is changed. I've tried to split mslug.ssb into 4 files with a size of 4MB each and named them to match the four 201-cX.cX files. This is where things don't really go the way I would've hoped, since none of them are a match for the files in the DotEmu ROM. I don't know where to find 201-s1.s1 either, since I'm not very knowledgeable about Neo Geo ROMs in general.

**Comment by jd213** [comment]
yeah, it says you can use POD Extract and then the MSC_extract_v2 from that blog to extract all the Metal Slug games except 5.

One of the comments says that it also works for the PSP version.

**Comment by RealRelativeEase** [comment]
#146 

I've run the blogpost through translation software, it says at the bottom that the extraction method doesn't work for SNK Arcade Classics Vol. 1 due to the way the compilation handles sound data.

The same blog also has a few entries about using UART to extract the ROMs from the Neo Geo Arcade Stick Pro, starting [here](https://web.archive.org/web/20220506235946/http://blog.livedoor.jp/scrap_a/archives/23139993.html). I don't have a Neo Geo Mini myself, but maybe someone can check whether the UART extraction method outlined in those posts works for the Neo Geo Mini as well?
